.github/workflows/prettier.yml

```
@@ -0,0 +1,65 @@
+          restore-keys: |
+            ${{ runner.os }}-node-modules-
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Run Prettier
+        run: yarn prettier --write .
+
+      - name: Check for changes
+        id: check_changes
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:27: Double quote to prevent globbing and word splitting

(shellcheck)